flyve-mdm-ios/Info.plist

```diff
@@ -39,5 +39,10 @@
 	<array>
 		<string>UIInterfaceOrientationPortrait</string>
 	</array>
+	<key>UIAppFonts</key>
+	<array>
+		<string>Roboto-Light.ttf</string>
+		<string>Roboto-Regular.ttf</string>
```

Comment:
I am using the default fonts of iOS for the screens.
San Francisco (SF)
The system font for macOS, iOS, watchOS, and tvOS.